Partial refactor of many .json and a repair to the existing methods t…

Revivalist-Dev opened the pull request:
…hat were replaced in the newest release candidate.

Etc. Etc, patches and more for the new candidate version.

Revivalist-Dev committed:
Partial refactor of many .json and a repair to the existing methods that were replaced in the newest release candidate.

Etc. Etc, patches and more for the new candidate version.

sekelsta commented:
Would you mind taking out the json formatting part of this? I actually prefer the json5 style without quoted keys.

Revivalist-Dev committed:
Revert "Partial refactor of many .json and a repair to the existing methods that were replaced in the newest release candidate."

This reverts commit cf3a26c7ea3c070355b0e8a25d0713b47fbf3be1.

Revivalist-Dev committed:
General updates for the Salt and Sands mod for the latest release candidate.

General updates to some of the new methods that have been updated for the latest release.
